README.md

```diff
@@ -203,6 +203,16 @@ Here's how to perform a delete:
 Newsletter::delete('rincewind@discworld.com');
 ```
 
+### Deleting subscribers permanently
+
+Delete all personally identifiable information related to a list member, and remove them from a list. This will make it impossible to re-import the list member.
```

Comment:
> This will make it impossible to re-import the list member.

If everything is deleted, wouldn't it be *possible* to re-import the list member?

Reply:
@riasvdv I had based the description on the Mailchimp docs found below (see the "Action" tab). I agree that it does make sense to be possible to re-import, but I thought it would be best to favour the official docs rather than potentially cause confusion.

https://developer.mailchimp.com/documentation/mailchimp/reference/lists/members/

Reply:
Okay, makes sense